tests/test_s3/test_s3.py

```diff
@@ -1630,19 +1630,26 @@ def test_delete_versioned_bucket_returns_meta():
         Bucket="blah", VersioningConfiguration={"Status": "Enabled"}
     )
 
-    client.put_object(Bucket="blah", Key="test1", Body=b"test1")
+    put_resp = client.put_object(Bucket="blah", Key="test1", Body=b"test1")
 
     # Delete the object
     del_resp = client.delete_object(Bucket="blah", Key="test1")
-    assert "DeleteMarker" not in del_resp
+    assert del_resp["DeleteMarker"] is False
```

Comment:
Is this correct? In the ticket you noted that `Deleting an object by key only will create a delete marker for it. The response will include the DeleteMarker flag set to True`